client/src/format.ts

```diff
@@ -35,6 +35,26 @@ export function formatCountyASMState(state: County.ASMState): string {
     }
 }
 
+function formatFileUploadStatus(countyStatus: DOS.CountyStatus): string {
+// here we are either really failed or pending
+    if (_.get(countyStatus, 'ballotManifest.result.errorMessage') === undefined &&
+        _.get(countyStatus, 'cvrExport.result.errorMessage') === undefined) {
+        return 'File Upload In Progress...';
+    } else {
+        return 'File Upload Failed';
+    }
+}
+
+function formatFileUploadStatusIndicator(countyStatus: DOS.CountyStatus): string {
+    // here we are either really failed or pending
+    if (_.get(countyStatus, 'ballotManifest.result.errorMessage') === undefined &&
+        _.get(countyStatus, 'cvrExport.result.errorMessage') === undefined) {
+        return 'status-indicator-in-progress';
```

Comment:
The `status-indicator-in-progress` is for statuses that indicate the audit is underway/in-progress, but since file uploads are before that, we should leave off this indicator class and only show the red circle if there's an error.